std/process.d

```diff
@@ -908,6 +908,19 @@ unittest // Reopening the standard streams (issue 13258)
     assert(lines == ["foo", "bar"]);
 }
 
+version (Windows)
```

Comment:
Why restrict the test to Windows? Isn't the point to make sure that the code does the same thing on all OSes?